task11/ALL.sql

```diff
@@ -0,0 +1,135 @@
+CREATE TABLE `posts` (
+  `idOfPost` int NOT NULL AUTO_INCREMENT,
+  `username` varchar(20) NOT NULL,
+  `description` varchar(280) DEFAULT NULL,
+  `createdAt` datetime DEFAULT NULL,
+  PRIMARY KEY (`idOfPost`),
+  KEY `fk_Posts_Users1_idx` (`username`),
+  CONSTRAINT `fk_Posts_Users1` FOREIGN KEY (`username`) REFERENCES `users` (`username`)
+) ENGINE=InnoDB AUTO_INCREMENT=42 DEFAULT CHARSET=utf16;
+/*!40101 SET character_set_client = @saved_cs_client */;
+
+--
+-- Dumping data for table `posts`
+--
+
+LOCK TABLES `posts` WRITE;
+/*!40000 ALTER TABLE `posts` DISABLE KEYS */;
+INSERT INTO `posts` VALUES (1,'incredible2','Hello','2019-06-24 22:47:41'),(2,'incredible6','Hello','2019-09-22 06:18:04'),(3,'incredible12','Hello','2020-02-08 01:05:05'),(4,'incredible20','Hello','2020-01-14 14:55:11'),(5,'incredible30','Hello','2020-05-09 14:24:51'),(6,'incredible42','Hello','2019-12-28 20:21:22'),(7,'incredible3','Hello','2019-10-26 11:06:47'),(8,'incredible19','Hello, it\'s me.','2019-07-13 19:47:52'),(9,'incredible37','Hello','2019-10-29 02:51:23'),(10,'incredible4','Hello','2020-05-08 07:21:44'),(11,'incredible26','Hello','2019-10-17 22:44:47'),(12,'incredible50','Hello','2019-11-09 21:23:19'),(13,'incredible23','Hello','2020-01-16 14:35:16'),(14,'incredible51','Hello','2020-05-09 14:40:46'),(15,'incredible28','Hello','2019-11-02 02:45:26'),(16,'incredible7','Hello','2019-09-05 21:06:50'),(17,'incredible41','Hello','2020-01-03 12:23:09'),(18,'incredible24','Hello','2020-05-12 08:46:20'),(19,'incredible9','Hello','2019-06-14 02:39:24'),(20,'incredible49','Hello','2019-06-20 18:38:58'),(21,'incredible4','Hello182','2019-08-16 00:27:58'),(22,'incredible4','Hello707','2019-10-13 00:22:08'),(23,'incredible4','Hello46','2019-11-10 11:57:09'),(24,'incredible4','Hello111','2019-12-07 10:14:22'),(25,'incredible4','Hello481','2019-12-15 02:38:40'),(26,'incredible4','Hello211','2020-01-05 17:41:38'),(27,'incredible4','Hello154','2019-06-07 13:53:03'),(28,'incredible4','Hello564','2020-01-23 12:15:11'),(29,'incredible4','Hello931','2019-07-05 20:30:55'),(30,'incredible4','Hello990','2020-04-25 00:08:16'),(31,'incredible5','Hello633','2019-07-10 09:20:03'),(32,'incredible5','Hello468','2019-09-27 18:55:54'),(33,'incredible5','Hello530','2019-12-27 12:39:01'),(34,'incredible5','Hello927. Yeeeee','2019-12-20 18:54:23'),(35,'incredible5','Hello549','2019-07-19 11:25:13'),(36,'incredible5','Hello616','2019-10-26 04:03:24'),(37,'incredible5','Hello682','2019-12-23 17:20:30'),(38,'incredible5','Hello208','2019-07-28 00:46:26'),(39,'incredible5','Hello940','2019-10-22 02:59:26'),(40,'incredible5','Hello999','2020-03-15 21:40:13'),(41,'iamnewuser','It\'s my post','2020-05-28 10:02:20');
+/*!40000 ALTER TABLE `posts` ENABLE KEYS */;
+UNLOCK TABLES;
+
+--
+-- Table structure for table `users`
+--
+
+DROP TABLE IF EXISTS `users`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!50503 SET character_set_client = utf8mb4 */;
+CREATE TABLE `users` (
```

Comment:
PK is missing in USERS table, which is bad. It will decrease performance of one-to-many and many-to-many queries greatly